training/ilab-wrapper/ilab

```diff
@@ -18,6 +18,32 @@ function verify_range() {
     fi
 }
 
+check_insights() {
+    if [[ -f /etc/insights-client/machine-id ]]; then
+        return
+    fi
+    if [[ -f /etc/ilab/skip-insights-validation ]]; then
+        return
+    fi
+    local ID
+    eval "$(grep ^ID= /etc/os-release)"
+    if [[ "$ID" != "rhel" ]]; then
+        return
+    fi
+    cat << EOF
+This host is not connected to Red Hat Insights.
+
+To connect this host to Red Hat Insights run the following command:
+sudo rhc connect --organization <org_id> --activation-key <your_activation_key>
+
+For more information managing your activation keys, please visit:
+https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys
```

Comment:
That doc is very thorough.  Possibly too much for a new user.  Proposing this change:

```This host is not connected to Red Hat Insights.
To connect this host to Red Hat Insights run the following command:
sudo rhc connect --organization <org_id> --activation-key <your_activation_key>

To generate an Activation Key: 
https://console.redhat.com/insights/connector/activation-keys (this page will also display your Organization ID).

For more information managing your activation keys, please visit:
https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys```

Reply:
I like your change

However, in the future we will provide a link to the official RHELAI documentation that is more concise, but for now, let's keep it as you suggest

Reply:
awesome thanks